### ISSUE TITLE
Postgres10 support

### ISSUE THREAD

**Issue by xarg** [wal-g/wal-g#49]
When I try to run ` wal-g backup-push` I get this error:

```
ERROR:  function pg_xlogfile_name_offset(pg_lsn) does not exist at character 23
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
STATEMENT:  SELECT file_name FROM pg_xlogfile_name_offset(pg_start_backup($1, true, false))
```

wal-g version: 0.1.2
PostgreSQL version: 10.1

Looks like it's something similar to this issue here: https://github.com/wal-e/wal-e/pull/339

**Comment by x4m** [comment]
Hi, Alex! Thanks for your interest in making WAL-G better.
This one was fixed at #25 
I think we should have a release soon with this and many other cool new features.

**Comment by xarg** [comment]
Ah, thanks @x4m didn't see that one. Closing this.
